<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LecteurVue</class>
 <widget class="QMainWindow" name="LecteurVue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>10</horstretch>
    <verstretch>10</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="l_titreDiapo">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TitreDiapo</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>128</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="b_arreterDiapo">
        <property name="text">
         <string>Arrêter le diaporama</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="b_lancerDiapo">
        <property name="text">
         <string>Lancer le diaporama</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>248</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="l_image">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>10</horstretch>
          <verstretch>10</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>cartesDisney/Disney_tapis.gif</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>278</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="l_titre">
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Titre</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>228</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="b_precedent">
        <property name="text">
         <string>Précédent</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>28</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="b_suivant">
        <property name="text">
         <string>Suivant</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="l_mode">
        <property name="text">
         <string>Mode</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>158</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Catégories</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="cb_cat1">
           <property name="text">
            <string>Categorie1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_cat2">
           <property name="text">
            <string>Categorie2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_cat3">
           <property name="text">
            <string>Categorie3</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>18</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="l_rangImage">
          <property name="text">
           <string>Rang</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="l_slash">
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="l_nbImage">
          <property name="text">
           <string>nbImage</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>454</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuParam_tres">
    <property name="title">
     <string>Paramètres</string>
    </property>
    <addaction name="a_chargerDiapo"/>
    <addaction name="a_changerVitesse"/>
    <addaction name="a_enleverDiapo"/>
   </widget>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="a_quitter"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="a_aPropos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuParam_tres"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="a_chargerDiapo">
   <property name="text">
    <string>Charger un diaporama</string>
   </property>
  </action>
  <action name="a_changerVitesse">
   <property name="text">
    <string>Changer la vitesse de défilement</string>
   </property>
  </action>
  <action name="a_enleverDiapo">
   <property name="text">
    <string>Enlever le diaporama</string>
   </property>
  </action>
  <action name="a_quitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="a_aPropos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
